### PR TITLE
Add GitHub CI workflow to validate the files in this repo, using these actions

### PR DESCRIPTION
**Pull request by ben-alkov**
"lint the linters"

### CODE REVIEW

File: shellcheck/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ RUN dnf -y --nodocs  --setopt=install_weak_deps=False \
 --disablerepo=fedora-cisco-openh264 \
 --disablerepo=fedora-modular \
 --disablerepo=updates-modular \
-install ShellCheck findutils file
+install ShellCheck findutils file; \
```
Comment (MartinBasti) [inline shellcheck/Dockerfile:7]: shouldn't be there `&&` instead `;` ?

Comment (MartinBasti) [reply shellcheck/Dockerfile:7]: everywhere in this PR

Comment (ben-alkov) [reply shellcheck/Dockerfile:7]: 🤔 
Not sure... local tests worked, but I'll check it again

Comment (ben-alkov) [reply shellcheck/Dockerfile]: Fixed

Comment (MartinBasti) [reply shellcheck/Dockerfile]: well, I mean, is there reason to continue when previous command fails?

```
$ test 0 -eq 1 ; echo "yes"
yes

$ test 0 -eq 1 && echo "yes"  
```

**Review by MartinBasti — APPROVED** [review]

**Review by chmeliik — APPROVED** [review]
:tada:

**Pull request merged**